### ISSUE TITLE
[Setup] Overview page and/or filter options

### ISSUE THREAD

**Issue by holanita** [Green-Software-Foundation/speakers#16]
@osamajandali I know this project is not a priority at the moment, but I wanted to log this here

These extra features could be useful to make the site more user-friendly
1) an overview page where we can see all the speakers in one place with some essential information (maybe [here](https://speakers.greensoftware.foundation/speakers/)?)
2) filter options so those who are looking for specific languages, regions, topics can find the right speakers more easily

Unsure if you guys have talked about this already. Let me know when you are ready to work on this and let's discuss then.

**Comment by holanita** [comment]
Just adding here some ideas about filters. I've been thinking about which fields we could use easily as filters. 

These look easy and relevant, although some speakers gave unique answers that might need to be tweaked a bit:
- location
- speaker experience (meetups, track talks, etc.)
- number of previous talks (ranges)
- coaching (yes/no)
- languages
Not sure if we want to focus on these too:
- company
- pronouns
- mapa?
- minority?
The rest is free text mostly, so I was thinking we could add
- a search field to look for any word / expressions in all the fields in across all the speaker profiles

**Comment by jawache** [comment]
@jawache chat to @osamajandali about:

- Can you use autogenerated pages for overview page.
- Can we use tags as the way to do a simple search with custom looking search results.

**Comment by holanita** [comment]
@jawache @osamajandali do you guys have any updates on this? please keep me posted. thank you.

**Comment by jawache** [comment]
@osamajandali / @holanita - I'm going to use this ticket to talk about just searching/filtering and this ticket to talk about a better landing/overview page (https://github.com/Green-Software-Foundation/speakers/issues/28)

I think the only way to get something that meets these requirements in the time we have is to use _existing_ features of the docusaurus platform rather than write something custom.

We can add tags to the speaker page and once those are added we can quickly show the people who are tagged with that tag using a link like so: 

https://patterns.greensoftware.foundation/tags/size-medium

So for each speaker let's pick a few items and put them as the speakers page in the front-matter as tags.

For example, let's take this speaker: https://github.com/Green-Software-Foundation/speakers/blob/main/docs/speakers/abhishek-gupta.mdx

If the top of the page had this content

```
---
sidebar_position: 1
tags:
- country:canada
- city:montreal
---
```

Then this page will show all the people who are in the country of Canada

https://speakers.greensoftware.foundation/tags/country-canada

There are a couple of things we can do @osamajandali you can create a sidebar item that is autogenerated from the tags, using something here: https://docusaurus.io/docs/sidebar/autogenerated

So the sidebar can have

Speakers >
....
By City >
- Montreal -> (On clicking this takes people to https://speakers.greensoftware.foundation/tags/country-canada)
By Country >
...

I hope that makes sense. Proper search and filtering would be better but I think this is something that at least gives people the ability to search by country/city which I think is going to be the main question. We can add proper search functionality later.

**Comment by zanete** [comment]
@holanita I understand that the tagging would need to be done before the functionality can work correctly, and that's something that is not going to be done by Osama himself, right? 

If my understanding is correct, then would you like a separate issue to explain / track the progress of tagging all speakers? 

From my point of view, it would be much easier for Osama to implement this kind of feature if there were a few speakers already tagged with all the possible tag types, so that should be done ahead of this issue.

cc @jawache

**Comment by holanita** [comment]
@zanete I don't know to be honest, but I'm happy to do the tagging myself if someone can explain to me what tags I can use

**Comment by zanete** [comment]
@jawache @holanita, @osamajandali does not believe this can be accomplished in the timeframe by COP, need to understand the importance of this (given how many speakers are at the website) and whether other things on @osamajandali plate can be moved to post-cop? 
Further risks is:
- It's an experimental approach, and without all the tags being in place it's hard to foresee how to implement the behaviour
- It will need time from You @jawache as it's expected there will be quite a bit of back-n-forth to achieve the desired outcome

**Comment by zanete** [comment]
for launch, @osamajandali will implement the tags for country and for going forward he will let @holanita know how to do it for future profiles.

**Comment by holanita** [comment]
@osamajandali @zanete the site looks amazing and the search feature works like a charm. simple but efficient. this is more than enough for now as we don't have that many speakers yet. I'm tempted to close this to avoid parking too many tasks. let me know if you don't agree.
